pocs/filterwheel/libefw.py

```diff
@@ -0,0 +1,181 @@
+import ctypes
+import enum
+
+from pocs.camera.sdk import AbstractSDKDriver
+from pocs.utils import error
+from pocs.utils.library import load_library
+
+
+class EFWDriver(AbstractSDKDriver):
+    # Because ZWO EFW library isn't linked properly have to manually load libudev
+    # in global mode first, otherwise get undefined symbol errors.
```

Comment:
Is that something on the fault of ZWO that might have a link we could include here for more information?

Reply:
ISTR there being discussion of the undefined udev symbols problem on the ZWO forums, and me getting the workaround from there. Good call about including a link, I'll see if I can find the relevant thread again.

Reply:
OK, don't seem to be able to find it in ZWO forums. Not sure how I came up with the workaround.

Reply:
No worries. I'll leave it up to you whether or not you think it a good enough idea to merit creating an issue out of it for the future in case it's found.